templates/api/knowledge-panels/environment/origins_of_ingredients.tt.json

```diff
@@ -12,6 +12,15 @@
         "icon_color_from_evaluation": true,
         "icon_url": "[% static_subdomain %]/images/icons/dist/public.svg",
     },
+        {
+            "element_type": "action",
+            "action_element": {
+                "html": `[% lang("action_add_origins") %]`,
+                "actions": [
+                    "add_origins",
+                ]
+            }
+        },
```

Comment:
You broke the json here ;-) (it should be included after "elements")
I'm fixing it.